### PR TITLE
Added basic script for jumpstart week 2

### PR DESCRIPTION
**Pull request by liammoyn**

### CODE REVIEW

**Comment by connernilsen** [comment]
Is this waiting on review?

**Comment by connernilsen** [comment]
Rebase with master 😠

**Review by sadafkh11 — APPROVED** [review]

File: jumpstart-week2/README.md [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+3. Use our example code to lecture about the basics of HTML
+- Bring up file for the homepage
+- Talk about format of HTML
+- Introduce tags, attributes
+- divs & spans
+
+4. Add some basic color and positioning CSS to make a basic layout
+- Have a stylesheet that is imported into homepage
+- Introduce className and id and css selector syntax
+- Have a CSS to define a layout already, have students use class names to insert elements into the layout
```
Comment (connernilsen) [inline jumpstart-week2/README.md]: Can you indent these subitems pls

File: jumpstart-week2/README.md [hunk 2]
```diff
@@ -0,0 +1,222 @@
...
+- Introduce the idea of node
+- npm install
+- npm run start
+- Open localhost, explain the idea
+
+3. Use our example code to lecture about the basics of HTML
+- Bring up file for the homepage
+- Talk about format of HTML
+- Introduce tags, attributes
+- divs & spans
```
Comment (connernilsen) [inline jumpstart-week2/README.md]: Capitalize

File: jumpstart-week2/README.md [hunk 3]
```diff
@@ -0,0 +1,222 @@
...
+
+CSS is used to control presentation, formatting, and layout. How does the page look?
+
+JavaScript is used to control the behavior of different elements. What can the page do?
+
+HTML, CSS, and Javascript are ever evolving technologies that were created around the year 2000, practically ancient in web development terms. They all have their own systems for adding new features and aren't always consistent with each other or even within themselves. Because of this, there are often dozens of ways to do the same thing with HTML, CSS and Javascript. This can be confusing to beginners or even experienced developers, but it's important to not get discouraged and to always keep an open mind to alternative approaches even if you don't need them.
+
+To learn more, [this blog](https://blog.hubspot.com/marketing/web-design-html-css-javascript) gives a good overview.
+
+> Side Note: Javascript is a programming language, HTML and CSS have their own syntax and can do some similar things, but they are *not* programming languages.
```
Comment (connernilsen) [inline jumpstart-week2/README.md]: [lmao](https://accodeing.com/blog/2015/css3-proven-to-be-turing-complete)

File: jumpstart-week2/README.md [hunk 4]
```diff
@@ -0,0 +1,222 @@
...
+HTML provides the basic structure of sites. What is on the page?
+
+CSS is used to control presentation, formatting, and layout. How does the page look?
+
+JavaScript is used to control the behavior of different elements. What can the page do?
+
+HTML, CSS, and Javascript are ever evolving technologies that were created around the year 2000, practically ancient in web development terms. They all have their own systems for adding new features and aren't always consistent with each other or even within themselves. Because of this, there are often dozens of ways to do the same thing with HTML, CSS and Javascript. This can be confusing to beginners or even experienced developers, but it's important to not get discouraged and to always keep an open mind to alternative approaches even if you don't need them.
+
+To learn more, [this blog](https://blog.hubspot.com/marketing/web-design-html-css-javascript) gives a good overview.
+
```
Comment (connernilsen) [inline jumpstart-week2/README.md]: Also, I don't know if this helps, but the best description I've heard of HTML/CSS/JS is
>HTML is like the bones
>CSS is like the skin
>JS is like the brain

And you can take this analogy as far as you want.

File: jumpstart-week2/README.md [hunk 5]
```diff
@@ -0,0 +1,222 @@
...
+`<element attributeName="attributeValue">`
+
+Attributes allow you to specify extra information about an element.
+
+#### How is HTML different than XML?
+
+Like we said earlier, HTML is a subset of XML. All HTML is also XML, not all XML is HTML.
+
+HTML defines several XML elements and gives guidelines on how they can be used together. There are a ton of different HTML elements that are defined and a lot of them aren't super useful so don't bother trying to learn all of them (google is your friend though).
```
Comment (connernilsen) [inline jumpstart-week2/README.md]: Maybe link to MDN and w3schools here

File: jumpstart-week2/README.md [hunk 6]
```diff
@@ -0,0 +1,222 @@
...
+#app {
+    text-align: center;
+    height: 400px;
+}
+
+.blog-post {
+    width: 100%;
+    height: 50px;
+    color: lightblue;
+}
```
Comment (connernilsen) [inline jumpstart-week2/README.md]: Maybe keep this example here, but add each example under the selector type description below.

File: jumpstart-week2/README.md [hunk 7]
```diff
@@ -0,0 +1,222 @@
...
+
+HTML defines several XML elements and gives guidelines on how they can be used together. There are a ton of different HTML elements that are defined and a lot of them aren't super useful so don't bother trying to learn all of them (google is your friend though).
+
+A couple of the important HTML elements to know are:
+
+- `<div>`: Defines a block element
+- `<span>`: Defines an inline element
+- `<p>`: Stands for paragraph, use this to wrap text
+
+For each defined element, HTML also defines a ton of different attributes that you can optionally add to an element. We'll see more of these in practice, and two important ones when we talk about CSS.
```
Comment (connernilsen) [inline jumpstart-week2/README.md]: Maybe talk about some of the more important element attributes here like `id`, `class`, and `href`.

File: jumpstart-week2/README.md [hunk 8]
```diff
@@ -0,0 +1,222 @@
...
+
+Like we said earlier, HTML is a subset of XML. All HTML is also XML, not all XML is HTML.
+
+HTML defines several XML elements and gives guidelines on how they can be used together. There are a ton of different HTML elements that are defined and a lot of them aren't super useful so don't bother trying to learn all of them (google is your friend though).
+
+A couple of the important HTML elements to know are:
+
+- `<div>`: Defines a block element
+- `<span>`: Defines an inline element
+- `<p>`: Stands for paragraph, use this to wrap text
```
Comment (connernilsen) [inline jumpstart-week2/README.md]: Maybe add `link` and `title` here too!

Comment (connernilsen) [reply jumpstart-week2/README.md]: Should we also talk about `head`, `body`, `<!DOCTYPE html>`, and `html` special html elements?

File: jumpstart-week2/README.md [hunk 9]
```diff
@@ -0,0 +1,222 @@
...
+To tackle sizing and positioning of elements, you first have to understand the "box model".
+
+Every HTML element has several boxes around it whose dimensions can be controlled through CSS. Every element has all of these boxes around it, though often these boxes can coincide.
+
+The innermost box is the "content box". This is the space that all the children of that element occupy. 
+
+The next box is the "border box". This includes all the content of the content box PLUS the border of the element.
+
+Then is the "padding box". This includes the border box PLUS some padding (empty space) that the element should be surrounded by.
+
```
Comment (connernilsen) [inline jumpstart-week2/README.md]: Throw in an image like box model

File: jumpstart-week2/README.md [hunk 10]
```diff
@@ -0,0 +1,222 @@
...
+For now, we will use the CSS we have already designed to fill in a layout. <Go into demo here>
+
+<Mention that CSS can be written in many different places>
+
+## Javascript
+
+Javascript allows you to add interactivity and complex functionality to your website. Javascript started as a simple scripting language designed to complete simple tasks on a webpage. Now we have Javascript frameworks like React that dominate modern web development. We'll see a lot more Javascript as we start working with React but for now we will stick to simple use cases.
+
+<Show boldIt & revertIt simple methods>
```
Comment (connernilsen) [inline jumpstart-week2/README.md]: Can you put those methods in here so that Jumpstarters looking at the docs can see this?

**Review by connernilsen — COMMENTED** [review]
I like it! Can you pull in master though and see #3 for how we should lay out this project with MkDocs? By the way, if you see a better way to lay it out, please let me know.

File: jumpstart-week2/README.md [hunk 11]
```diff
@@ -0,0 +1,222 @@
...
+
+XML defines a syntax (a set of rules) to express data. There are 3 main aspects of XML.
+
+A tag (or element) is the basic unit.
+
+Example: `<element>`
+
+Every tag has a name which is the first word following immediately after the opening bracket (`<`). Every tag also has to have a closing tag after it. A closing tag is simply `</element>` where `element` is just the name of the tag repeated again. 
+
+Between the opening and closing tags are the second aspect of XML, children. Between an element's two tags is a list of either other XML tags or text. Every item in this list is a child of the element that is surrounding them. Of course, an XML element that is the child of an element can also have its own children. You'll often here the term "depth" of an XML element to refer to the number of parents that element has.
```
Comment (connernilsen) [inline jumpstart-week2/README.md]:
```suggestion
Between the opening and closing tags are the second aspect of XML, children. Between an element's two tags is a list of either other XML tags or text. Every item in this list is a child of the element that is surrounding them. Of course, an XML element that is the child of an element can also have its own children. You'll often hear the term "depth" of an XML element to refer to the number of parents that element has.
```

Comment (connernilsen) [reply jumpstart-week2/README.md]: typo :)

File: jumpstart-week2/README.md (hunk 8 above)
Comment (jacktonina) [reply jumpstart-week2/README.md]: I'm with Conner, I think if we're briefly introducing these few tags - it can't hurt to throw in most of the big ones. From someone who's not super familiar with HTML either, would be good to include as many as are relevant.

File: jumpstart-week2/README.md (hunk 9 above)
Comment (jacktonina) [reply jumpstart-week2/README.md]: Picture is helpful, I'd include

File: jumpstart-week2/README.md [hunk 12]
```diff
@@ -0,0 +1,222 @@
...
+
+To tackle sizing and positioning of elements, you first have to understand the "box model".
+
+Every HTML element has several boxes around it whose dimensions can be controlled through CSS. Every element has all of these boxes around it, though often these boxes can coincide.
+
+The innermost box is the "content box". This is the space that all the children of that element occupy. 
+
+The next box is the "border box". This includes all the content of the content box PLUS the border of the element.
+
+Then is the "padding box". This includes the border box PLUS some padding (empty space) that the element should be surrounded by.
```
Comment (jacktonina) [inline jumpstart-week2/README.md]: 'Then is the "Padding Box"'

File: README.md [hunk 13]
```diff
@@ -0,0 +1,65 @@
+# C4C Dev Docs
+
+C4C Dev Docs holds all technical documentation for our applications and projects. The goal of this site is to bring existing information together to a single, searchable, and pleasant UI. 
+
+As of now, we've copied over the LLB and SFTT API specs. There is some work to update those to be more user-friendly for this site, but I'm very happy with how MkDocs took our existing Markdown files and created this site so effortlessly. 
+
+Other things to expect to see here: general repo setup guide for our major projects, DevOps/Deployment overview for LLB/SFTT cloud pipelines, Jumpstart course material...
```
Comment (jtavera235) [inline README.md]: This is probably small but instead of saying repo we should say that repo is short for repository beforehand or explain it here.

File: README.md [hunk 14]
```diff
@@ -0,0 +1,65 @@
...
+
+
+## Local Installation
+
+This repo uses two Python packages, [MkDocs](https://www.mkdocs.org/) and [Material for MkDocs](https://squidfunk.github.io/mkdocs-material/getting-started/), to generate a static documentation site for C4C repos. Pages are written in Markdown.
```
Comment (jtavera235) [inline README.md]: Explain what Markdown is?

File: jumpstart-week2/README.md (hunk 2 above)
Comment (liammoyn) [reply jumpstart-week2/README.md]: I believe these should stay uncapitalized

File: README.md [hunk 15]
```diff
@@ -0,0 +1,65 @@
...
+
+Point your browser to [localhost:8000](http://localhost:8000/)
+
+### Building your site
+When you're finished editing, you can build a static site from your Markdown files with:
+```
+mkdocs build
+```
+
+The contents of this directory make up your project documentation. There's no need for operating a database or server, as it is completely self-contained. The site can be hosted on GitHub Pages, GitLab Pages, a CDN of your choice or your private web space.
```
Comment (jtavera235) [inline README.md]: Explain what CDN is?

File: README.md [hunk 16]
```diff
@@ -0,0 +1,65 @@
...
+
+### Adding a Page (local copy of repo)
+
+Add a Markdown file to the `docs/` directory. 
+
+Edit the navigation in `mkdocs.yml`. The structure is well documented [here](https://code-4-community.github.io/c4c-dev-docs/). 
+
+To publish your changes, create a PR.
```
Comment (jtavera235) [inline README.md]: Explain that PR means Pull request and explain what that means?

File: jumpstart-week2/README.md (hunk 4 above)
Comment (wwohlers) [reply jumpstart-week2/README.md]: I actually really like the description we have here, I think it's concise and descriptive.

File: jumpstart-week2/README.md (hunk 8 above)
Comment (wwohlers) [reply jumpstart-week2/README.md]: Adding on to this, I think it could be helpful to briefly define "inline" and "block," otherwise "defines a block element" won't mean much to people who aren't already familiar with HTML. Or we can describe divs/spans another way.

File: jumpstart-week2/README.md (hunk 12 above)
Comment (wwohlers) [reply jumpstart-week2/README.md]: Not sure if this is what Jack is suggesting, but doesn't the padding box come before the border box, as you have in the diagram? The diagram & text following it seem to contradict

File: jumpstart-week2/README.md [hunk 17]
```diff
@@ -0,0 +1,222 @@
...
+
+> Side Note: pixels don't actually refer to the pixels on your screen, they're a set length that is meant to be consistent across devices and browsers.
+
+##
+
+Defining layouts can be one of the hardest parts of mastering CSS and figuring out how to position an element just right can frustrate even seasoned developers. This is challenging because a good layout has to be adaptable to many different browser sizes that a user may have.
+
+The two prominent CSS systems for creating flexible layouts are called 'flexbox' and 'grid'. Flexbox is great for items that should be grouped together but don't need to have a particular size. Grid is great when there are very clear boundaries for where elements should be. These layouts can be used together as well for more complicated layout patterns. To learn more about these layouts and how to implement themselves you can visit [here](https://css-tricks.com/snippets/css/a-guide-to-flexbox/) for flexbox and [here](https://css-tricks.com/snippets/css/complete-guide-grid/) for grid.
+
```
Comment (wwohlers) [inline jumpstart-week2/README.md]: Should we also introduce the `display` and `position` properties? These are fundamental to understanding CSS layout

File: docs/workshops2019/index.md [hunk 18]
```diff
@@ -0,0 +1,34 @@
+# Workshop Series 2019
```
Comment (jacktonina) [inline docs/workshops2019/index.md:1]: 2019

File: README.md (hunk 15 above)
Comment (jacktonina) [reply README.md]: ^

**Review by jacktonina — APPROVED** [review]

File: docs/lesson-plans/week-2.md [hunk 19]
```diff
@@ -0,0 +1,241 @@
...
+
+HTML decides what is on the page, CSS decides what the page looks like, and Javascript decides what the page can do.
+
+HTML, CSS, and Javascript are ever evolving technologies that were created around the year 2000, practically ancient in web development terms. They all have their own systems for adding new features and aren't always consistent with each other or even within themselves. Because of this, there are often dozens of ways to do the same thing with HTML, CSS and Javascript. This can be confusing to beginners or even experienced developers, but it's important to not get discouraged and to always keep an open mind to alternative approaches even if you don't need them.
+
+To learn more, [this blog](https://blog.hubspot.com/marketing/web-design-html-css-javascript) gives a good overview of some of the basics of the three technologies.
+
+> Side Note: Javascript is a programming language, HTML and CSS have their own syntax and can do some similar things, but they are *not* programming languages.
+
+</>
```
Comment (jtavera235) [inline docs/lesson-plans/week-2.md]: Is this suppose to be here?

File: README.md (hunk 13 above)
Comment (maxsebso) [reply README.md]: second this

File: README.md (hunk 14 above)
Comment (maxsebso) [reply README.md]: second this

File: README.md (hunk 15 above)
Comment (maxsebso) [reply README.md]: ^

File: README.md [hunk 20]
```diff
@@ -0,0 +1,65 @@
...
+When you're finished editing, you can build a static site from your Markdown files with:
+```
+mkdocs build
+```
+
+The contents of this directory make up your project documentation. There's no need for operating a database or server, as it is completely self-contained. The site can be hosted on GitHub Pages, GitLab Pages, a CDN of your choice or your private web space.
+
+### Adding a Page (local copy of repo)
+
+Add a Markdown file to the `docs/` directory.
```
Comment (maxsebso) [inline README.md]: Maybe a little more explanation. Like Add a file named "some-file-name.md"

File: README.md [hunk 21]
```diff
@@ -0,0 +1,65 @@
...
+mkdocs build
+```
+
+The contents of this directory make up your project documentation. There's no need for operating a database or server, as it is completely self-contained. The site can be hosted on GitHub Pages, GitLab Pages, a CDN of your choice or your private web space.
+
+### Adding a Page (local copy of repo)
+
+Add a Markdown file to the `docs/` directory. 
+
+Edit the navigation in `mkdocs.yml`. The structure is well documented [here](https://code-4-community.github.io/c4c-dev-docs/).
```
Comment (maxsebso) [inline README.md]: Again maybe some more detail

File: docs/lesson-plans/week-2.md [hunk 22]
```diff
@@ -0,0 +1,241 @@
...
+          Alice
+        </third-grandchild>
+    </third-child>
+</parent>
+```
+
+> Side note: Often it'll be the case that an XML element has zero children. You can still write out both the opening and closing tags with nothing between them but XML also has a shorthand that saves some time retyping the closing tag:
+> `<element></element` === `<element />`
+
+The third aspect of XML is *attributes*. Attributes live within the opening tag of XML elements. Attributes have a name and a value, both of which are just plain strings (words). The attribute value will always be wrapped in double quotes or single quotes.
```
Comment (jtavera235) [inline docs/lesson-plans/week-2.md]: I think this sounds a little cleaner - "Attributes have a name and value..."

Comment (jtavera235) [inline docs/lesson-plans/week-2.md]: "in double or single quotes"

File: docs/lesson-plans/week-2.md [hunk 23]
```diff
@@ -0,0 +1,241 @@
...
+
+```xml
+<element attributeName="attributeValue" />
+```
+
+Attributes allow you to specify extra information about an element.
+
+#### How is HTML different than XML?
+
+Like we said earlier, HTML is a subset of XML. All HTML is also XML, but not all XML is HTML.
```
Comment (jtavera235) [inline docs/lesson-plans/week-2.md]: "All HTML is XML..."

File: docs/lesson-plans/week-2.md [hunk 24]
```diff
@@ -0,0 +1,241 @@
...
+
+HTML defines several XML elements and gives guidelines on how they can be used together. There are a ton of different HTML elements that are defined and a lot of them aren't super useful so don't bother trying to learn all of them (google is your friend though).
```
Comment (jtavera235) [inline docs/lesson-plans/week-2.md]: I would say  "so don't worry about learning them all..."

File: docs/lesson-plans/week-2.md [hunk 25]
```diff
@@ -0,0 +1,241 @@
...
+- `<p>`: Stands for paragraph, use this to wrap text
+
+For each defined element, HTML also defines a ton of different attributes that you can optionally add to an element. We'll see more of these in practice, and two important ones when we talk about CSS.
+
+In addition, for a file to be considered an HTML file, there are a couple of standard patterns you need to follow. But, because we're using React for everything, React takes care of all of that for us, so we won't cover it here. See [here](http://csis.pace.edu/~wolf/HTML/htmlnotepad.htm) for details.
+
+
+## CSS
+
+CCS is the standard for adding styling to your web page. HTML can exist without CSS but CSS cannot do anything without HTML.
```
Comment (jtavera235) [inline docs/lesson-plans/week-2.md]: CSS not CCS

File: docs/lesson-plans/week-2.md [hunk 26]
```diff
@@ -0,0 +1,241 @@
...
+
+> Side Note: Browsers are essentially parsers of HTML, CSS, and Javascript. The reason the development of HTML, CSS, and Javascript are so distributed / hard to follow is because for a new feature to be added, browsers have to update their engines to support the feature. You'll often see compatibility charts for newer features that show which browser versions support the feature and which do not (this is why you hear web developers curse internet explorer so much).
+
+## HTML
+
+HTML is a subset of XML. We'll first describe XML in general and then talk about the rules that make HTML different.
+
+#### What is XML?
+
+XML defines a syntax (a set of rules) to express data. There are 3 main aspects of XML.
```
Comment (maxsebso) [inline docs/lesson-plans/week-2.md]: Maybe add that XML stands for Extensible Markup Language

File: docs/lesson-plans/week-2.md [hunk 27]
```diff
@@ -0,0 +1,241 @@
...
+    - Have a stylesheet that is imported into homepage
+    - Introduce className and id and css selector syntax
+    - Have a CSS to define a layout already, have students use class names to insert elements into the layout
+
+5. Walk through building out a simple post component
+
+
+## What is HTML, CSS and Javascript?
+
+**HTML** provides the basic structure of sites.
```
Comment (maxsebso) [inline docs/lesson-plans/week-2.md]: Maybe add that HTML stands for Hypertext Markup Language

File: docs/lesson-plans/week-2.md [hunk 28]
```diff
@@ -0,0 +1,241 @@
...
+
+**CSS** is used to control presentation, formatting, and layout.
```
Comment (maxsebso) [inline docs/lesson-plans/week-2.md]: Maybe add that CSS stands for Cascading Style sheet

File: docs/lesson-plans/week-2.md [hunk 29]
```diff
@@ -0,0 +1,241 @@
...
+![Box Model](./img/box-model.png)
+
+Every HTML element has several boxes around it whose dimensions can be controlled through CSS. Every element has all of these boxes around it, though often these boxes can coincide.
+
+The innermost box is the *content box*. This is the space that all the children of that element occupy. 
+
+The next box is the *border box*. This includes all the content of the content box PLUS the border of the element.
+
+Then is the *padding box*. This includes the border box PLUS some padding (empty space) that the element should be surrounded by.
+
```
Comment (maxsebso) [inline docs/lesson-plans/week-2.md]: Add that we can also create our own boxes to couple other elements: the <div>

File: docs/workshops2019/index.md [hunk 30]
```diff
@@ -0,0 +1,34 @@
...
+## Source Control with Git
+
+
+This workshop goes over the basics of source control and how git can be used to effectively manage project versioning.
+
+GitPresentation.pptx is the intro slideshow that goes over the basics of source control and git.
+
+demonotes.md are the speaker notes used throughout the demo including helpful learning resources at the end. The speaker notes also include all code that was written throughout the demo.
+
+[Source Control with Git](git/index.md)
```
Comment (maxsebso) [inline docs/workshops2019/index.md]: Prob should reformat this and reword to be used in Jumpstart. Also we should add more about git documented

**Review by maxsebso — CHANGES_REQUESTED** [review]

**Comment by maxsebso** [comment]
Solid job so far

File: docs/lesson-plans/week-2.md (hunk 19 above)
Comment (liammoyn) [reply docs/lesson-plans/week-2.md]: Yeah used so that the two different side notes don't blend into each other

File: jumpstart-week2/README.md (hunk 2 above)
Comment (connernilsen) [reply jumpstart-week2/README.md]: ok sure

File: jumpstart-week2/README.md (hunk 4 above)
Comment (connernilsen) [reply jumpstart-week2/README.md]: ok!

File: docs/lesson-plans/week-2.md [hunk 31]
```diff
@@ -0,0 +1,239 @@
...
+    text-align: center;
+    height: 400px;
+}
+
+
+```
+
+#### Selectors
+
+CSS provides about a dozen ways of defining the elements you want a rule to apply to. We'll cover 3 of the most standard. You can generally get by with these for most use cases, but if a situation requires more complex selection logic you should be aware that other methods exist and you can google them.
```
Comment (kiminlee2000225) [inline docs/lesson-plans/week-2.md]: "We'll cover 3 of the most standard." was a bit confusing when I first read it (might just be me). Possibly add something like "We'll cover 3 of the most standard methods to identify elements to apply the desired CSS styling to."

**Review by jtavera235 — APPROVED** [review]
LGTM

File: docs/lesson-plans/week-2.md [hunk 32]
```diff
@@ -0,0 +1,239 @@
...
+
+There is also regular `width` and `height` properties. By default, these define the size of the content box, but you can override this behavior so that they refer to the size of the content plus the border by setting `box-sizing: border-box;` on an element. This is common practice now because it often makes development simpler.
+
+For now, we'll only deal with lengths in units of pixels or `px`. You can also define lengths with [different units](https://css-tricks.com/the-lengths-of-css/) including units that are relative to the font size, relative to the size of the browser, and relative to the parent element.
+
+> Side Note: pixels don't actually refer to the pixels on your screen, they're a set length that is meant to be consistent across devices and browsers.
+
+##
+
+Defining layouts can be one of the hardest parts of mastering CSS and figuring out how to position an element just right can frustrate even seasoned developers. This is challenging because a good layout has to be adaptable to many different browser sizes and resolutions that a user may have.
```
Comment (kiminlee2000225) [inline docs/lesson-plans/week-2.md]: Mention how an example for layout adaptability is the need for your software to be suitable for mobile (and for the different sizes of mobile screens) using CSS.

File: docs/jumpstart/week-2.md [hunk 33]
```diff
@@ -0,0 +1,239 @@
...
+
+HTML decides what is on the page, CSS decides what the page looks like, and Javascript decides what the page can do.
+
+HTML, CSS, and Javascript are ever evolving technologies that were created around the year 2000, practically ancient in web development terms. They all have their own systems for adding new features and aren't always consistent with each other or even within themselves. Because of this, there are often dozens of ways to do the same thing with HTML, CSS and Javascript. This can be confusing to beginners or even experienced developers, but it's important to not get discouraged and to always keep an open mind to alternative approaches even if you don't need them.
+
+To learn more, [this blog](https://blog.hubspot.com/marketing/web-design-html-css-javascript) gives a good overview of some of the basics of the three technologies.
+
+> Side Note: Javascript is a programming language, HTML and CSS have their own syntax and can do some similar things, but they are *not* programming languages.
+
+</>
```
Comment (connernilsen) [inline docs/jumpstart/week-2.md:33]: What is this? Does it render the way you want it to on MkDocs?

Comment (liammoyn) [reply docs/jumpstart/week-2.md:33]: It ensures the side notes are split the correct way

**Review by maxsebso — APPROVED** [review]
LGI
